### ISSUE TITLE
Vex dialogues not closing properly on Chrome OS X latest

### ISSUE THREAD

**Issue by cimocimocimo** [HubSpot/vex#135]
Chrome 44.0.2403.157

If you click on the page as the modal is fading out you can prevent the modal from fully closing. If you continue to click on the page the 'vex-open' class won't be removed from the body and the modal will capture all the click events preventing the user from interacting with the page.

Seems to be chrome only. I tested in latest versions of FF and Safari any they didn't show this behaviour. If you stop clicking around on the page the modal will close and the 'vex-open' class is removed from the body after a couple seconds.

Steps to reproduce:
1) open vex dialogue in chrome.
2) close dialogue
3) click on the page as the modal is closing and continue to click the page.

**Comment by cimocimocimo** [comment]
I'm thinking this is probably a bug with Chrome but thought I would report it anyway incase others are having this issue.

**Comment by cimocimocimo** [comment]
Seems the issue is a bug with Chrome latest stable. I downloaded Canary and it seems the bug has been fixed there. Clicking on the page while the animation is running seems to prevent or reset the 'animationend' event from firing. If that event doesn't fire then the rest of the close code doesn't run.

Since this looks to be fixed in the Version 47.0.2497.0 canary (64-bit) and in Version 46.0.2490.6 dev (64-bit) I wouldn't worry too much about this. IF someone really does need this fixed I'd suggest a monkeypatch to use setTimeout() to manually call the needed close functions and trigger vexClose and vexAfterClose events.
